boltz/api.go

```diff
@@ -111,6 +111,7 @@ type NodeInfo struct {
 type Nodes = map[string]map[string]NodeInfo
 
 type SwapStatusResponse struct {
+	Id               string `json:"id"`
```

Comment:
To avoid trying to use the `id` in responses to GET requests don't add it here. But create a new types that extends this one that is only used in the websocket